numba/core/typing/npydecl.py

```diff
@@ -439,7 +439,10 @@ def register_number_classes(register_global):
 # -----------------------------------------------------------------------------
 # Numpy array constructors
 
-def _parse_shape(shape):
+def parse_shape(shape):
```

Comment:
API change to public, seems safe WRT https://github.com/numba/numba/issues/5214

Reply:
there is no change suggested here, right?

Reply:
Correct, no change. Was just noting that it was being changed and that a check was made for immediate impact.